docs/messagecontext.md

```diff
@@ -40,6 +40,8 @@ Returns **[string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Refer
 
 # FTLList
 
+# FTLList
+
```

Comment:
I'd know what this is, some artifact of `make docs`.  I'd like to fix `make docs` in general so let's ignore this for now.